ProtoZBuffer/CommandLineExecuter.cs

```diff
@@ -46,6 +46,11 @@ private static bool Execute(IOption option)
 
         static bool CommonVisit<T>(AbstractOption option) where T : AbstractGenerator, new()
         {
+            if (string.IsNullOrEmpty(option.Namespace) || string.IsNullOrWhiteSpace(option.Namespace))
```

Comment:
string.IsNullOrWhiteSpace also tests for empty strings, cf MSDN : Indicates whether a specified string is null, empty, or consists only of white-space characters.